v2/tools/generator/internal/codegen/pipeline/report_resource_versions.go

```diff
@@ -183,6 +186,20 @@ func (report *ResourceVersionsReport) createTable(
 		return astmodel.ComparePathAndVersion(right.PackageReference.PackagePath(), left.PackageReference.PackagePath())
 	})
 
+	samplesMap := make(map[string]string)
+	err := filepath.WalkDir(report.samplesPath, func(filePath string, d fs.DirEntry, err error) error {
+		if !d.IsDir() && filepath.Base(filepath.Dir(filePath)) != "refs" {
+			sampleLink := filepath.Join(report.rootUrl, filePath)
+			sampleFile := filepath.Base(filePath)
+			samplesMap[sampleFile] = sampleLink
+		}
+
+		return nil
+	})
+	if err != nil {
+		return nil, errors.Wrap(err, "walking through samples dir")
```

Comment:
Tell the user which samples folder was being walked through

```suggestion
		return nil, errors.Wrapf(err, "walking through samples directory %s", report.samplesPath)
```